client-src/elements/chromedash-guide-editall-page.js

```diff
@@ -98,16 +102,34 @@ export class ChromedashGuideEditallPage extends LitElement {
     setupScrollToHash(this);
   }
 
-  handleFormSubmit(event, hiddenTokenField) {
-    event.preventDefault();
+  handleFormSubmit(e, hiddenTokenField) {
+    e.preventDefault();
+    const submitBody = formatFeatureChanges(this.fieldValues, this.featureId);
 
     // get the XSRF token and update it if it's expired before submission
     window.csClient.ensureTokenIsValid().then(() => {
       hiddenTokenField.value = window.csClient.token;
-      event.target.submit();
+      return csClient.updateFeature(submitBody);
```

Comment:
Previous 2 lines might no longer be needed.

Reply:
I'll open a subsequent PR and remove these calls 👍